checks/test_program.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
```

Comment:
A really nice touch to make sure that the header compiles and that the data is printable.